src/Bridge/Symfony/Bundle/Resources/config/metadata.xml

```diff
@@ -116,24 +116,12 @@
 
         <!-- Cache -->
 
-        <service id="api_platform.metadata.resource.cache.array" class="Symfony\Component\Cache\Adapter\ArrayAdapter" public="false">
-            <argument>0</argument>
-            <argument>false</argument>
+        <service id="api_platform.metadata.resource.cache" parent="cache.system" public="false">
```

Comment:
I think we can leverage the "system" cache, but if my understanding of the code is correct, these will be used in the `dev` environment as well... (Still, we should be safe because of the nonce invalidation.)